coriolisclient/cli/endpoint_instances.py

```diff
@@ -34,7 +34,7 @@ class EndpointInstanceFormatter(formatter.EntityFormatter):
 
     def _get_formatted_data(self, obj):
         data = (obj.id,
-                obj.name,
+                obj.instance_name,
```

Comment:
Not all providers return ‘instance_name’ afaik so we should ideally do an ‘obj.get(“instance_name”)’ for safety.